src/components/Modals/NewSemester.vue

```diff
@@ -2,63 +2,141 @@
+          <div class="newSemester-dropdown-placeholder season-arrow" id="season-arrow"></div>
+        </div>
+        <div class="newSemester-dropdown-content season-content" id="season-content">
+          <div class="newSemester-dropdown-content-item" id="fall" v-on:click="selectSeason('fall')">
+            🍂 Fall
+          </div>
+          <div class="newSemester-dropdown-content-item" id="spring" v-on:click="selectSeason('spring')">
+            🌸 Spring
+          </div>
+          <div class="newSemester-dropdown-content-item" id="summer" v-on:click="selectSeason('summer')">
+            ☀️ Summer
+          </div>
+          <div class="newSemester-dropdown-content-item" id="winter" v-on:click="selectSeason('winter')">
+            ❄️ Winter
+          </div>
+        </div>
+      </div>
     </div>
     <div class="newSemester-section newSemester-year">
       <label class="newSemester-label" for="year">{{ yearText }}</label>
-      <select class="newSemester-select" id="year" v-html="yearOptions" />
+      <div class="newSemester-select" id="year">
+        <div class="newSemester-dropdown-placeholder year-wrapper" v-on:click="showHideYearContent">
+          <div class="newSemester-dropdown-placeholder year-placeholder" id="year-placeholder">{{ yearPlaceholder }}</div>
+          <div class="newSemester-dropdown-placeholder year-arrow" id="year-arrow"></div>
+        </div>
+        <div class="newSemester-dropdown-content year-content" id="year-content">
+          <div v-for="year in years" :key="year" :id = year class="newSemester-dropdown-content-item" v-on:click="selectYear(year)">{{year}}</div>
+        </div>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
-// TODO: gray out initial dropdown options
 export default {
+  data() {
+    // years
+    const currentYear = new Date().getFullYear();
+    const years = [];
+    let startYear = currentYear - 10;
+    while (startYear <= currentYear + 10) {
+      years.push(startYear);
+      startYear += 1;
+    }
+    years.map(String);
+
+    return {
+      seasonPlaceholder: 'Select One',
+      yearPlaceholder: (new Date()).getFullYear(),
+      years
+    };
+  },
   computed: {
     typeText() {
       return 'Type';
     },
     yearText() {
       return 'Year';
+    }
+  },
+  methods: {
+    showHideContent(contentID, boxID, arrowID) {
+      const content = document.getElementById(contentID);
+      const box = document.getElementById(boxID);
+      const arrow = document.getElementById(arrowID);
+
+      const contentShown = content.style.display === 'block';
+
+      if (contentShown) { // clicked box when content shown. So then hide content
+        content.style.display = 'none';
+        box.style.borderColor = '#C4C4C4';
+        arrow.style.borderTopColor = '#C4C4C4';
+      } else {
+        content.style.display = 'block';
+        box.style.borderColor = '#32A0F2';
+        arrow.style.borderTopColor = '#32A0F2';
+      }
+    },
```

Comment:
Do not use raw DOM manipulation to set style. 
This doc shows how to do it with Vue's style binding: https://vuejs.org/v2/guide/class-and-style.html